src/cvodes/cvodes_nls_stg.c

```diff
@@ -119,45 +116,41 @@ int CVodeSetNonlinearSolverSensStg(void *cvode_mem, SUNNonlinearSolver NLS)
   retval = SUNNonlinSolSetConvTestFn(cv_mem->NLSstg, cvNlsConvTestSensStg,
                                      cvode_mem);
   if (retval != CV_SUCCESS) {
```

Comment:
Missing a call to do error checking:
```suggestion
  SUNCheckCallNoRet(retval);
  if (retval != CV_SUCCESS) {
```